Content.Shared/Access/Systems/AccessReaderSystem.cs

```diff
@@ -236,7 +236,8 @@ public bool FindAccessItemsInventory(EntityUid uid, out HashSet<EntityUid> items
             }
 
             // maybe its inside an inventory slot?
-            if (_inventorySystem.TryGetSlotEntity(uid, "id", out var idUid))
+            if (_inventorySystem.TryGetSlotEntity(uid, "id", out var idUid) ||
+                _inventorySystem.TryGetSlotEntity(uid, "belt", out idUid))
```

Comment:
This seems like a really dumb way for this to work, instead of like, an inventory relayed event, or even granting a component...
Oh well, that's not your problem :(